.eslintrc.js

```diff
@@ -20,6 +20,18 @@ module.exports = {
   extends: ['@elastic/eslint-config-kibana', 'plugin:@elastic/eui/recommended'],
   rules: {
     // "@osd/eslint/require-license-header": "off"
+    '@osd/eslint/no-restricted-paths': [
+      'error',
+      {
+        basePath: __dirname,
+        zones: [
+          {
+            target: ['(public|server)/**/*'],
```

Comment:
Can these be expanded, maybe *.ts/*.js files under any path?

Reply:
@peternied Sorry I did not see this comment last week. This was modeled after eslint rules from OSD core. Example: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.eslintrc.js#L320-L334

I was trying to add this rule to OSD Core instead of this repo to enforce it across all dashboards plugins, but I was not ultimately able to figure out how to.